### PR TITLE
[ISSUE #2556] Lite pull consumer support msg trace

### PR DESCRIPTION
**Pull request by zhangjidi2016**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

#2556

### CODE REVIEW

**Comment by zhangjidi2016** [comment]
I have resubmitted this PR, please help to review it @duhenglucky  @zongtanghu

File: client/src/main/java/org/apache/rocketmq/client/consumer/DefaultLitePullConsumer.java [hunk 1]
```diff
@@ -188,6 +200,57 @@ public DefaultLitePullConsumer(final String consumerGroup, RPCHook rpcHook) {
...
+
+    /**
+     * Constructor specifying consumer group, enabled msg trace flag and customized trace topic name.
+     *
+     * @param consumerGroup        Consumer group.
+     * @param enableMsgTrace       Switch flag instance for message trace.
+     * @param customizedTraceTopic The name value of message trace topic.If you don't config,you can use the default
+     *                             trace topic name.
+     */
+    public DefaultLitePullConsumer(final String consumerGroup, boolean enableMsgTrace,
```
Comment (duhenglucky) [inline client/src/main/java/org/apache/rocketmq/client/consumer/DefaultLitePullConsumer.java:221]: How about removing this constructor? Maybe use the set method to set related configurations would be a better choice.

Comment (zhangjidi2016) [reply client/src/main/java/org/apache/rocketmq/client/consumer/DefaultLitePullConsumer.java:221]: This is a good suggestion, but this submission still wants to be consistent with DefaultMQPushConsumer and DefaultMQProducer. If necessary, a PR can be proposed later to rectify the problem of too many constructors

**Comment by zhangjidi2016** [comment]
image

image

**Review by duhenglucky — APPROVED** [review]

**Comment by zongtanghu** [comment]
LGTM

**Comment by cserwen** [comment]
@zhangjidi2016 Why clientHost was removed? Do we not consider compatibility？

**Pull request merged**